scanpipe/pipes/d2d.py

```diff
@@ -114,6 +125,55 @@ def get_best_path_matches(to_resource, matches):
     return matches
 
 
+def load_ecosystem_config(pipeline, options):
+    """
+    Add ecosystem specific configurations for each ecosystem selected
+    as `options` to the `pipeline`.
```

Comment:
We need to provide more details about what's actually happening when "adding" a config.

Reply:
Added, thanks!